src/shared/bootspec.c

```diff
@@ -485,8 +484,6 @@ int boot_loader_read_conf(BootConfig *config, FILE *file, const char *path) {
                         r = free_and_strdup(&config->auto_firmware, p);
                 else if (streq(field, "console-mode"))
                         r = free_and_strdup(&config->console_mode, p);
-                else if (streq(field, "random-seed-mode"))
```

Comment:
This will print a warning now, no? I think it should be more graceful, and put out an info message about the deprecation first

Reply:
Okay, I'll do that.

Reply:
Done.